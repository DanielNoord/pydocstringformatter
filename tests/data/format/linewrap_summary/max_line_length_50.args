--linewrap-full-docstring
--max-line-length=50
