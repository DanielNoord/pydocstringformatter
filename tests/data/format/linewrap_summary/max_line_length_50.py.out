def func():
    """A very long line that needs to be wrapped
    especially this sentence.
    """

def func():
    """Event multi line docstrings need to be
    wrapped.

    This description is way too long. It definitely needs to be wrapped.
    """

# Regression for bug found in pylint
# We should re-add the quotes to line length if they will never be on the first line.
class LinesChunk:
    """The LinesChunk object computes and
    store.
    """
